alpine/src/test/scala/io/delta/alpine/internal/DeltaLogSuite.scala

```diff
@@ -241,36 +238,58 @@ class DeltaLogSuite extends QueryTest
 
 // TODO test("handle corrupted '_last_checkpoint' file") { }
 
-  test("paths should be canonicalized - normal and special characters") {
```

Comment:
Why do you need to split this test?

Reply:
1. it's split in Delta OSS code (I merged it before, but then changed my mind)
2. a benefit of it being split: if the test fails, what case caused it to fail? the normal characters? or the special characters? having it split lets you easily identify that